clang/test/SemaSYCL/parallel_for_wrapper_attr.cpp

```diff
@@ -0,0 +1,54 @@
+// RUN: %clang_cc1 %s -fsyntax-only -ast-dump -fsycl -fsycl-is-device -triple spir64 | FileCheck %s
```

Comment:
I don't know how we typically organize test, but in community, tests for `-ast-dump` typically live in the `AST` directory.